mirage/fixtures/versions.js

```diff
@@ -0,0 +1,6 @@
+export default {
```

Comment:
I think we dont need this file since the version is hardcoded in `mirage/config.js`.